autopush/tests/test_web_validation.py

```diff
@@ -158,13 +158,13 @@ def _make_fut(self):
         schema.context["log"] = Mock()
         return schema
 
-    def _make_test_data(self, headers=None, body="", path_args=None,
-                        path_kwargs=None, arguments=None):
+    def _make_test_data(self, headers=None, body="", args=None, kwargs=None,
+                        arguments=None):
```

Comment:
Hrm. While it feels like a nit, `args` vs. `arguments` might be a bit confusing. Granted, `path_args` wasn't terribly great either, but at least it reflected what it was used for. 

Since it's kinda/sorta unused. I suppose we could just drop the `args` for now.